topologies/datacenter/files/MenuOptions.yaml

```diff
@@ -58,25 +64,35 @@ labconfiglets:
       - "Spine2-BGP-Lab"
     leaf1:
       - "Leaf1-VXLAN-Lab"
+      - "VLANs"
     leaf2:
       - "Leaf2-VXLAN-Lab"
+      - "VLANs"
     leaf3:
       - "Leaf3-VXLAN-Lab"
+      - "VLANs"
     leaf4:
       - "Leaf4-VXLAN-Lab"
+      - "VLANs"
+    cvx01:
+      - "cvx01-Controller"
   l2evpn:
     spine1:
       - "Spine1-L2EVPN-Lab"
     spine2:
       - "Spine2-L2EVPN-Lab"
     leaf1:
       - "Leaf1-L2EVPN-Lab"
+      - "VLANs"
     leaf2:
       - "Leaf2-L2EVPN-Lab"
+      - "VLANs"
     leaf3:
       - "Leaf3-L2EVPN-Lab"
```

Comment:
Remove `Leaf3-L2EVPN-Lab` configlet from leaf3.  This configure Leaf3 all the way for the L2EVPN lab.